pkg/api/handlers/compat/containers.go

```diff
@@ -44,11 +45,24 @@ func RemoveContainer(w http.ResponseWriter, r *http.Request) {
 	runtime := r.Context().Value("runtime").(*libpod.Runtime)
 	name := utils.GetName(r)
 	con, err := runtime.LookupContainer(name)
-	if err != nil {
-		utils.ContainerNotFound(w, name, err)
-		return
-	}
+	if err != nil && errors.Cause(err) == define.ErrNoSuchCtr {
+		// Failed to get container. If force is specified, get the container's ID
+		// and evict it
+		if !query.Force {
+			utils.ContainerNotFound(w, name, err)
```

Comment:
idk, but might it be worth a debug statement here as a bread crumb?

Reply:
This mirrors the behavior from the ABI code path. I am looking to bring them together not more different.